frontend/src/individual_certificate.tsx

```diff
@@ -585,26 +637,40 @@ export function IndividualCertificate({ id }: { id: string }) {
               color="primary"
               disabled={isUserGood ? false : true}
               onClick={() => {
-                issueCertificate(id, certificateData);
+                action == "issue"
```

Comment:
This is probably fine (and short, which is nice), but I have a slight preference for e.g. `switch` as it gives me a cuddly feeling of more clearly implying multivalue and thus less risk of missing changes required if adding new `action` values.

But I don't have that much typescript experience, so the best case would be to use (non-confusing) language features if they can help enforce/flag things like this.

Reply:
good point! changed it.